src/Api.Rest/Common/Authentication/IAuthenticationHandler.cs

```diff
@@ -0,0 +1,42 @@
+#region copyright
+
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+/* Carl Zeiss Industrielle Messtechnik GmbH        */
+/* Softwaresystem PiWeb                            */
+/* (c) Carl Zeiss 2016                             */

```

Comment:
For some new files (not all) the copyright year is not correct, could you please adapt them? For older files it is ok to keep the original year, but for new ones it should be the current year.